5-flexbox/assets/styles.css

```
@@ -0,0 +1,57 @@
+*{
```

* will be applied for all the elements.
```suggestion
body {
```